src/styles/sass/features/_base.scss

```diff
@@ -0,0 +1,71 @@
+@import '../abstracts/';
+
+.features {
+    width: 100%;
+    background-color: $color-black;
+    color: $color-white;
+    padding-top: 20px;
+    padding-bottom: 20px;
+
+    .features__header {
+        text-align: center;
+        font-size: 1.2em;
+        font-weight: bold;
+        padding: 20px 0 20px 0;
+    }
+
+    .features__container {
+        @include flexAbsoluteCenter;
+        display: grid;
+        grid-template-columns: 80%;
+        grid-template-rows: repeat(3, 250px);
+        grid-auto-flow: row;
+
+        .feature {
+            @include flexAbsoluteCenter;
+            flex-direction: column;
+
+            &>* {
+                margin-bottom: 20px;
+            }
+
+            .feature__image {
+                width: 40px;
+                height: 40px;
+                padding: 10px;
+                border-radius: 50%;
+                background-color: $color-white;
+            }
+
+            .feature__description {
+                font-weight: bold;
+                text-align: center;
+            }
+
+            .feature__text {
+                text-align: center;
+            }
+        }
+    }
+}
+
+@media(min-width: 768px) {
```

Comment:
Fajną opcją do tworzenia różnych wersji "mobilnych" jest coś takiego tj. odpowiednie mixyny, które można wygodnie wykorzystać: https://dev.to/thebuildguy/easy-to-write-media-queries-using-sass-mixins-1p2f